src/mono/wasm/build/WasmApp.targets

```diff
@@ -151,6 +172,8 @@
 
       <WasmNativeAsset Include="$(MicrosoftNetCoreAppRuntimePackRidNativeDir)$(WasmIcuDataFileName)" Condition="'$(InvariantGlobalization)' != 'true'" />
       <WasmNativeAsset Include="$(MicrosoftNetCoreAppRuntimePackRidNativeDir)dotnet.timezones.blat" />
+
+      <WasmFilesToIncludeInFileSystem Include="$(TargetDir)publish\runtimeconfig.bin" />
```

Comment:
Move this to the end of `_WasmGenerateRuntimeConfig` target, since it belongs there. And it gets added only if we run that target.

Reply:
Make sense!